bin/oneoff/data_fix/update_last_seen_school_info_interstitial

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env ruby
+
+#
+# Script to update the last_seen_school_info_interstitial in users.
+#
+
+require_relative '../../../dashboard/config/environment'
+require 'optparse'
+
+CDO.log = Logger.new(STDOUT)
+ActiveRecord::Base.record_timestamps = false
+
+options = {interstitial_days: nil, csv_file: '/tmp/fix_bad_school_info_in_users.201710112304.csv'}
```

Comment:
nice :) Good thinking to record the results in a CSV

Reply:
And good thing it was still on the production-console :) I think a next version of this type of script should include an additional S3 archive (with expiration).